packages/automerge-repo/src/DocHandle.ts

```diff
@@ -328,6 +328,19 @@ export class DocHandle<T> //
     return this.#doc
   }
 
+  /**
+   * Returns the current "heads" of the document, akin to a git commit.
+   * This precisely defines the state of a document. In the event of multiple heads,
+   * there are unmerged concurrent changes to the document.
```

Comment:
This suggests that the document state is unresolved if there are multiple heads. My impression was that Automerge has no "merge commits", so having multiple heads just means that there were concurrent changes at some point, and there haven't been any changes since then; but the concurrent changes are all materialized in the document, so there are no differences between 1 head and 2+ heads that we need to care about. Is that understanding correct?

Reply:
Yes. I guess the word "unmerged" is misleading here. I'll  just remove it: it's accurate without.